Jenkinsfile

```diff
@@ -1,7 +1,10 @@
 #!groovy
 
 if (JENKINS_URL == 'https://ci.jenkins.io/') {
-    buildPlugin()
+    buildPlugin(
+      platforms: ['linux'],
+      tests: [skip: true]
```

Comment:
@batmat We are skipping tests, this is intentional? Won't this result in to releasing blueocean incrementally with test failures?